lib/module.js

```diff
@@ -6,21 +7,23 @@ const deepMerge = require('deepmerge')
 const logger = require('consola').withScope('nuxt:sentry')
 
 module.exports = function sentry(moduleOptions) {
+  const publicPath = this.options.build.publicPath
   const defaults = {
     dsn: process.env.SENTRY_DSN || false,
     disabled: process.env.SENTRY_DISABLED || false,
     disableClientSide: process.env.SENTRY_DISABLE_CLIENT_SIDE || false,
+    disableServerSide: process.env.SENTRY_DISABLE_SERVER_SIDE || false,
     publishRelease: process.env.SENTRY_PUBLISH_RELEASE || false,
     config: {
       environment: this.options.dev ? 'development' : 'production'
     },
     webpackConfig: {
-      include: ['.nuxt/dist/client'],
+      include: ['.nuxt/dist/client', '.nuxt/dist/server'],
```

Comment:
~> `.nuxt/dist` or maybe `{buildDir}/dist`?

Reply:
@pi0 is there a way to get buildDir relative path ? this.options.buildDir will have an absolute path and sentry-webpack-plugin wants here relative path

Reply:
I would suggest something like this: `path.relative(rootDir, buildDir)`